build/FindPulseAudio.cmake

```diff
@@ -20,55 +20,21 @@ if (PULSEAUDIO_LIBRARIES AND PULSEAUDIO_INCLUDE_DIRS)
 else (PULSEAUDIO_LIBRARIES AND PULSEAUDIO_INCLUDE_DIRS)
   # use pkg-config to get the directories and then use these values
   # in the FIND_PATH() and FIND_LIBRARY() calls
-  if (${CMAKE_MAJOR_VERSION} EQUAL 2 AND ${CMAKE_MINOR_VERSION} EQUAL 4)
+  #  if (${CMAKE_MAJOR_VERSION} EQUAL 2 AND ${CMAKE_MINOR_VERSION} EQUAL 4)
     include(UsePkgConfig)
-    pkgconfig(libpulse _PULSEAUDIO_INCLUDEDIR _PULSEAUDIO_LIBDIR _PULSEAUDIO_LDFLAGS _PULSEAUDIO_CFLAGS)
```

Comment:
Interestingly, this line does NOT have version parameter.  Maybe pkg config's parameter format changes?